pandas/core/base.py

```diff
@@ -309,6 +309,15 @@ def transpose(self: _T, *args, **kwargs) -> _T:
     def shape(self) -> Shape:
         """
         Return a tuple of the shape of the underlying data.
+
+        Examples
+        --------
+        >>> s = pd.Series([1, 2, 3])
+        >>> s.shape
+        (3,)
+
+        >>> pd.Series([1, 2, 3]).shape
+        (3,)
```

Comment:
thanks for your PR! I think just the first one should be fine, no need to repeat the example